bosh_cli/lib/cli/commands/ssh.rb

```diff
@@ -6,7 +6,7 @@ module Command
     class Ssh < Base
 
       # bosh ssh
-      usage 'ssh'
+      usage 'ssh <job>/<index>'
```

Comment:
Yes.  `bosh ssh jobname/0`, `bosh ssh jobname/0 ls /tmp` and `bosh ssh jobname 0` all work